client/src/containers/Register/styles.css

```diff
@@ -0,0 +1,68 @@
+.register-container {
+    text-align: center;
+}
+
+.register-container .form_element {
+    margin-top: 20px;
+}
+
+.register-container h2 {
+    font-size: 33px;
+    font-weight: 300;
+    color: #02bcd4;
+}
+
+.register-container input {
+    font-size: 20px;
+    border-radius: 3px;
+    border: 1px solid #ebebeb;
+    box-sizing: border-box;
+    padding: 11px 31px;
+    background: #f7f7f7;
+    font-weight: 300;
+    width: 260px;
+}
+
+.register-container button {
+    font-size: 20px;
+    padding: 10px 52px;
+    background: #02bcd4;
+    color: #fff;
+    font-weight: 300;
+    border-radius: 3px;
+    border: none;
+    margin-top: 20px;
+    width: 260px;
+}
+
+.register-container .error {
+    color: #FF5722;
+    margin-top: 10px;
+    font-weight: 600;
+}
+
+.register-container .field-error {
+    border: 2px solid #FF5722;
+    color: #FF5722;
+}
+
+.register-container .current_users {
+    width: 90%;
+    max-width: 500px;
+    margin: auto;
+    margin-top: 50px;
+}
+
+.register-container .current_users h3 {
+    font-size: 24px;
+    font-weight: 300;
+    color: orange;
+}
+
+.register-container .current_users th, td {
```

Comment:
As I understand, you want align content inside **th** and **td** which are inside `.current_users`. If I'm right, you should fix your code like this:
```
.register-container .current_users th,
.register-container .current_users td {
    /* styles */
}
```
Now all content inside **td** in all application are centered :)

Reply:
Ahaha, fuck... )))
Thank you! :D